views/js/buckaroo.admin.js

```diff
@@ -37,7 +37,11 @@ $(document).ready(function () {
         }
     });
 
-    $(".toggle-switch").each(function() {
+    $('.buckaroo-toggle-switch').change(function() {
```

Comment:
why do you got 2 change events registered to the same event? why not merge them together ?